### ISSUE TITLE
loading xml file

### ISSUE THREAD

**Issue by covchamp** [primaeval/script.tvguide.fullscreen#124]
Hi been using your add for years  and been great but recently for some reason i keep getting a bad status code when it try's to download my xml file
http://covchump.com/rom/roguide.xml
if i load the file from local disc it works fine but when i try from url i keep getting bad status code.

Any ideas?

Thanks

**Comment by covchamp** [comment]
any help on this? in the kodi log it says file does not exist but if you click the link it clearly does

**Comment by jonjo40** [comment]
have you tried HTTPS and HTTP and also with www. and without?

**Comment by covchamp** [comment]
yeah just tried all variations and still no joy. doing my head in lol

**Comment by covchamp** [comment]
![error](https://user-images.githubusercontent.com/31183755/61406382-a6b5aa00-a8d3-11e9-86a9-cf152b7e5006.jpg)

**Comment by covchamp** [comment]
the error log from when i run tvguide

**Comment by covchamp** [comment]
this was taken when i was playing with the address, changed back now to whats posted above but same error log comes

**Comment by ghost** [comment]
@covchamp I'm a programist for m-TVGuide, but I could maybe help you out if you send me the complete log.
